api/src/main/java/ca/bc/gov/educ/api/school/config/SchoolAPIMVCConfig.java

```diff
@@ -0,0 +1,43 @@
+package ca.bc.gov.educ.api.school.config;
+
+import lombok.AccessLevel;
+import lombok.Getter;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.InterceptorRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+/**
+ * The type School api mvc config.
+ *
+ * @author Om
+ */
+@Configuration
+public class SchoolAPIMVCConfig implements WebMvcConfigurer {
+
+  /**
+   * The Pen reg api interceptor.
```

Comment:
Some comments are referring to different api in some of these files

Reply:
Let me delete it